### ISSUE TITLE
Is there a code can run and output some answer?

### ISSUE THREAD

**Issue by wa008** [karpathy/llm.c#203]
Newbie for this project, I total agree that this project is helpful to Newbie to learn about LLM.

I wonder if there is a code that can build a final product like ChatGPT, it can get some input from user and ouput some answer.

**Comment by dagelf** [comment]
Yes, many other projects, but not this project yet. This project is for building the models and so far just small models that came before, so that we can make the newer models faster.
